charts/metrics-server/README.md

```diff
@@ -33,9 +33,14 @@ The following table lists the configurable parameters of the _Metrics Server_ ch
 | `serviceAccount.create`              | If `true`, create a new service account.                                                                                                                                                                                                                         | `true`                                     |
 | `serviceAccount.annotations`         | Annotations to add to the service account.                                                                                                                                                                                                                       | `{}`                                       |
 | `serviceAccount.name`                | Service account to be used. If not set and `serviceAccount.create` is `true`, a name is generated using the full name template.                                                                                                                                  | `nil`                                      |
+| `serviceAccount.secrets`             | The list of secrets mountable by this service account. See https://kubernetes.io/docs/reference/labels-annotations-taints/#enforce-mountable-secrets                                                                                                             | `[]`                                       |
 | `rbac.create`                        | If `true`, create the RBAC resources.                                                                                                                                                                                                                            | `true`                                     |
 | `rbac.pspEnabled`                    | If `true`, create a pod security policy resource.                                                                                                                                                                                                                | `false`                                    |
 | `apiService.create`                  | If `true`, create the `v1beta1.metrics.k8s.io` API service. You typically want this enabled! If you disable API service creation you have to manage it outside of this chart for e.g horizontal pod autoscaling to work with this release.                       | `true`                                     |
+| `apiService.annotations`             | Annotations to add to the API service                                                                                                                                                                                                                            | `{}`                                       |
```

Comment:
Do we need to add `.` at the end?

Reply:
I'll add it, we just need to be stricter in general about the PRs to improve the documentation quality and consistency.